src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -150,6 +151,51 @@ async function getAdditionalFacilityInfo(futureAppointments, useV2 = false) {
   return facilityData;
 }
 
+/**
+ * Wrapper function to retrieve facility information from the appointment
+ * record when using the v2 api or make api call to retrieve facility
+ * information.
+ *
+ * @param {*} appointments
+ * @param {*} dispatch
+ * @param {*} featureFacilitiesServiceV2
+ */
+async function getAdditionalFacilityInfoV2(
+  appointments,
+  dispatch,
+  featureFacilitiesServiceV2,
+) {
+  let facilityData;
+  if (featureFacilitiesServiceV2) {
+    // Facility information included with v2 appointment api call.
+    const facilities = appointments
+      .map(
+        appt =>
+          appt.vaos.apiData.location
+            ? { ...appt.vaos.apiData.location.attributes }
+            : null,
+      )
+      .filter(n => n);
+    facilityData = transformFacilitiesV2(facilities);
```

Comment:
What do you think about calling transformFacilityV2 from the `transformVAOSAppointment` function in services/appointments/transformers.v2.js? It keeps the transformation code in services layer and out of Redux, which I think is more consistent with the application overall.